test/basic.html

```diff
@@ -236,6 +247,59 @@
 
   });
 
+  suite('resetting', function () {
+    test('form has the right default values', function() {
+      var form = fixture('FormForResetting');
+
+      assert.equal(form._customElements.length, 1);
+      assert.equal(form.elements.length, 3);
+
+      assert.equal(form._customElementsInitialValues.length, 1);
+      assert.equal(form._customElementsInitialValues[0], 'zag');
```

Comment:
It would be nice if this could somehow be expressed as an introspective public API. If a test needs to access protected or private API to check that something is working, there is a chance our users will need to access that API for the same reason.